deploy-manage/monitor/logging-configuration/enabling-elasticsearch-audit-logs.md

```diff
@@ -1,6 +1,11 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/enable-audit-logging.html
+applies:
+  hosted: all
+  ece: all
+  eck: all
```

Comment:
is this correct even though we have another page for orchestrated deployments?

Reply:
Yes it is correct, we will have to refine it to explain that the objective of the page (which is to configure an elasticsearch setting) can be accomplished for all deployment types, as all deployment types support the change of this specific setting (of course each orchestrator has its own method to `configure elasticsearch`), but the content of the page is valid for all deployment types.

Reply:
I now see what you mean with the `other` page, you are right. Technically it is still valid, as a page showing simply an elasticsearch feature or setting that can be configured in orchestrated deployments is valid for orchestrators (of course assuming the user knows how to work with their orchestrator).

Anyway I'll be working pretty soon with all the `audit logging` and I have very good ideas to merge all the content in a nice and effective way. Let's review this in our next PR ;)